modin/core/io/pickle/pickle_dispatcher.py

```diff
@@ -47,9 +47,9 @@ def _read(cls, filepath_or_buffer, **kwargs):
         The number of partitions is equal to the number of input files.
         """
         if not (isinstance(filepath_or_buffer, str) and "*" in filepath_or_buffer):
-            warnings.warn("Defaulting to Modin core implementation")
             return cls.single_worker_read(
                 filepath_or_buffer,
+                "Defaulting to Modin core implementation",
```

Comment:
should we be saying Modin core? Does this mean anything to the user, or would it be better to just say "Modin".

Reply:
Spoke with Rehan today; it seems this case occurs when a buffer or a file path with a wildcard (*) in it is passed in. I've updated the message accordingly.